#!/usr/bin/env python
PACKAGE="sim_clock"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("real_time_factor", double_t, 1, "real time factor", 1.0, 0.001, 10.0)
gen.add("play_pause", bool_t, 1, "play/pause", True)
gen.add("single_step", bool_t, 1, "single step if paused", False)
gen.add("dt", double_t, 1, "time delta", 0.01, -1.0, 1.0)

exit(gen.generate(PACKAGE, "sim_clock", "Clock"))
